documentation/docs/04-layouts.md

```diff
@@ -76,132 +76,120 @@ Some parts of your app might need something other than the default layout. For t
+└ four/+page@foo.svelte        # ❌ page has `@foo`, but +layout-foo is not 'in scope'
 ```
 
 #### Inheritance chains
 
-Layouts can themselves choose to inherit from named layouts, from the same directory or a parent directory. For example, `x/y/__layout@root.svelte` is the default layout for `/x/y` (meaning `/x/y/one`, `/x/y/two` and `/x/y/three` all inherit from it) because it has no name. Because it specifies `@root`, it will inherit directly from the nearest `__layout-root.svelte`, skipping `__layout.svelte` and `x/__layout.svelte`.
+Layouts can themselves choose to inherit from named layouts, from the same directory or a parent directory. For example, `x/y/+layout@root.svelte` is the default layout for `/x/y` (meaning `/x/y/one`, `/x/y/two` and `/x/y/three` all inherit from it) because it has no name. Because it specifies `@root`, it will inherit directly from the nearest `+layout-root.svelte`, skipping `+layout.svelte` and `x/+layout.svelte`.
 
 ```
 src/routes/
 ├ x/
 │ ├ y/
-│ │ ├ __layout@root.svelte
-│ │ ├ one.svelte
-│ │ ├ two.svelte
-│ │ └ three.svelte
-│ └ __layout.svelte
-├ __layout.svelte
-└ __layout-root.svelte
+│ │ ├ +layout@root.svelte
+│ │ ├ one/+page.svelte
+│ │ ├ two/+page.svelte
+│ │ └ three/+page.svelte
+│ └ +layout.svelte
+├ +layout.svelte
+└ +layout-root.svelte
 ```
 
-> In the case where `__layout-root.svelte` contains a lone `<slot />`, this effectively means we're able to 'reset' to a blank layout for any page or nested layout in the app by adding `@root`.
+> In the case where `+layout-root.svelte` contains a lone `<slot />`, this effectively means we're able to 'reset' to a blank layout for any page or nested layout in the app by adding `@root`.
 
-If no parent is specified, a layout will inherit from the nearest default (i.e. unnamed) layout _above_ it in the tree. In some cases, it's helpful for a named layout to inherit from a default layout _alongside_ it in the tree, such as `__layout-root.svelte` inheriting from `__layout.svelte`. We can do this by explicitly specifying `@default`, allowing `/x/y/one` and siblings to use the app's default layout without using `x/__layout.svelte`:
+If no parent is specified, a layout will inherit from the nearest default (i.e. unnamed) layout _above_ it in the tree. In some cases, it's helpful for a named layout to inherit from a default layout _alongside_ it in the tree, such as `+layout-root.svelte` inheriting from `+layout.svelte`. We can do this by explicitly specifying `@default`, allowing `/x/y/one` and siblings to use the app's default layout without using `x/+layout.svelte`:
 
 ```diff
 src/routes/
 ├ x/
 │ ├ y/
-│ │ ├ __layout@root.svelte
-│ │ ├ one.svelte
-│ │ ├ two.svelte
-│ │ └ three.svelte
-│ └ __layout.svelte
-├ __layout.svelte
--└ __layout-root.svelte
-+└ __layout-root@default.svelte
+│ │ ├ +layout@root.svelte
+│ │ ├ one/+page.svelte
+│ │ ├ two/+page.svelte
+│ │ └ three/+page.svelte
+│ └ +layout.svelte
+├ +layout.svelte
+-└ +layout-root.svelte
```

Comment:
will this render correctly? Without `-` symbol moving intendation of `└`? In VSCode preview it shows wrong.

Reply:
we'll find out when this is merged into #5778 and the docs are able to build